app/features/build_runner/build_runner.rb

```diff
@@ -118,11 +121,13 @@ def complete_run(start_time:, artifact_paths: [])
 
     def checkout_sha
       if git_fork_config
-        repo.switch_to_fork(clone_url: git_fork_config.clone_url,
```

Comment:
bye bye obj-c style byyyyeeeeee

Reply:
Oh, is that the obj-c style for coding things? I haven't used obj-c before. The reason I changed the style was to preserve horizontal whitespace.

Reply:
yeah, that's ok, I think I like the change to the other style anyway 😄